docs/deployment/config_descriptions.md

```diff
@@ -4,51 +4,93 @@ title: Configuration Descriptions
 hide_title: true
 ---
 
-# IAM Configuration
-
-## iam-deployment-name
-
-The name (e.g. dev) of the the 'deployment' - this key tells kes which configuration set (in addition to the default values) to use when creating the cloud formation template[^1]
-
-## prefix
-
-This value will prefix CloudFormation-created IAM resources and permissions. **The `stackName` used in the [app](deployment/deployment-readme#configure-and-deploy-the-cumulus-stack) deployment must start with this prefix or the deployment will not work correctly.**
-
-## stackName
-
-The name of this iam stack in CloudFormation (e.g. <prefix>-iam).
-
-**The cumulus stack name must start with `<prefix>`** [^2]
-
-## buckets
-
-The buckets created in the [Create S3 Buckets](#create-s3-buckets) step. Buckets are defined in the config.yml with a key, name, and type. Types should be one of: internal, public, private, or protected. Multiple buckets of each type can be configured. A key is used for the buckets to allow for swapping out the bucket names easily.
-
-## useNgapPermissionBoundary
+# Cumulus Configuration
+
+## Overview
+
+### config.yml Explained
+
+| field | default     | description
+| ----- | ----------- | -----------
+| prefix | (required) | the name used as a prefix in all aws resources
```

Comment:
Can we move all the required items to the top of this table?

Reply:
Would prefer not to as it requires breaking up listings for certain objects such as `cmr`